conn_test.go

```diff
@@ -981,6 +985,38 @@ func testBrokers(t *testing.T, conn *Conn) {
 	}
 }
 
+func testReadTopics(t *testing.T, conn *Conn) {
+	time.Sleep(2 * time.Second)
```

Comment:
What is this sleep playing at?

Reply:
gonna remove that. Tests were failing locally because of my local kafka state.